open-api/rest-catalog-open-api.yaml

```diff
@@ -2763,7 +2763,8 @@ components:
         
         ## General Configurations
         
-        - `token`: Authorization bearer token to use for table requests if OAuth2 security is enabled 
+        - `token`: Authorization bearer token to use for table requests if OAuth2 security is enabled
+        - `expires-at-ms`: if present, specifies timestamp in milliseconds when credentials for storage(AWS S3/Azure/), specified in config would expire
```

Comment:
Please specify the reference point in time for `expires-at-ms`. A number of milliseconds by themselves are not sufficient to deduce the expiry moment.

Reply:
I'd also suggest using ISO 8601 for the value format.

Reply:
@dimas-b , Thanks for the review, will update it similar to https://github.com/apache/iceberg/pull/10722